cmd/switch/switch.go

```diff
@@ -0,0 +1,16 @@
+package _switch
```

Comment:
@caiordjesus @GuilhermBrSp 
Ss **switch** is a Golang reserved word, it can't be used as a package name. The underscore was automatically added by the IDE. WDYT?

Reply:
I learned to always put an _ on these situations. It's a lot easier to read in comparison to any other words or acronyms!